### ISSUE TITLE
ESP32 SPI DMP

### ISSUE THREAD

**Issue by Lynxu** [kriswiner/MPU-9250#373]
Hello @kriswiner do you know how can i get working dmp on a ESP32 on spi. I'm preety new to stuff like this. I got stuck on a project where i have three MPU9250 and need the quaternions for them.

**Comment by kriswiner** [comment]
No, I don't use the DMP on the MPU9250. I prefer to do the sensor fusion on
the host MCU.

**Comment by Lynxu** [comment]
Do you have something that could help me to get the quaternions? Any information is needed right now.

**Comment by kriswiner** [comment]
This:  https://github.com/kriswiner/ESP32/tree/master/MPU9250_MS5637

and this: https://github.com/kriswiner/MPU9250/tree/master/Dual_MPU9250

**Comment by Lynxu** [comment]
@kriswiner which of your sketches think would work the best for esp32

**Comment by kriswiner** [comment]
Maybe this one:
https://github.com/kriswiner/ESP32/tree/master/MPU9250_MS5637

**Comment by Lynxu** [comment]
@kriswiner did you made something that could work on spi too?

**Comment by kriswiner** [comment]
No, but this should be a straightforward change to the read/writeByte
functions.
